ansible/roles/couchdb/templates/couch-restarter.sh.j2

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
sh doesn't support the regular expression syntax for the wildcard matching, so the if condition was always evaluating to false and so the email was being sent every minute.  the emails are going to keep trickling in due to the backlog so i changed the subject in the message so we can tell which ones are real.